### DISCUSSION TITLE
Many databases vs used_id column performance

### DISCUSSION THREAD

**Discussion by shedar** [apache/doris#27513] [A - General / Q&A] [answered]
Is there an overhead in having many databases? E.g. for a hundred of independant users with a same data schema, would it be better from performance and memory requirements perspective to have DB per user or to have user_id column in each table with ROW POLICY for each user?

**Comment by yiguolei** [comment, marked as answer]
hundred of databases do not have any overhead. You could use in such way.

**Reply by shedar** [reply]
Awesome. Thank you for the quick answer.
